web-animations/interfaces/AnimationEffectTiming/easing.html

```diff
@@ -40,17 +40,49 @@
 });
 
 test(function(t) {
+  var invalidEasings = [
+    '',
+    'test',
+    'initial',
+    'inherit',
+    'unset',
+    'steps(3, nowhere)',
+    'steps(-3, end)',
+    'cubic-bezier(0.1, 0, 4, 0.4)',
+    'function (a){return a}',
+    'function (x){return x}',
+    'function(x, y){return 0.3}',
+    '7',
```

Comment:
Nit: I think this one should be moved further up the list. It seems a bit strange having it at the bottom after all the function-def and function-call items.